docs/local-overlay.yaml

```diff
@@ -1,18 +1,17 @@
 description: |
   A local overlay to deploy the charms from this repo. This is meant for developers.
-  The appropriate steps to use this would be to :
+  The appropriate steps to use this would be to:
   1) pull the repo locally
   2) cd metallb-operator
-  2) build the charms with `tox -e build` 
+  2) build the charms with `make charms` 
   3) deploy with the overlay: 
-      `juju deploy ./bundle.yaml --overlay ./docs/local-overlay.yaml`
-bundle: kubernetes
+      `juju deploy ./bundle --overlay ./docs/local-overlay.yaml`
 applications:
   metallb-controller:
-    charm: ../charms/metallb-controller/build/metallb-controller.charm
+    charm: ./build/metallb-controller.charm
     resources:
       metallb-controller-image: 'metallb/controller:v0.9.3'
   metallb-speaker:
-    charm: ../charms/metallb-speaker/build/metallb-speaker.charm
+    charm: ./build/metallb-speaker.charm
```

Comment:
should be ../build/metallb-speaker.charm

Reply:
I didn't realize that paths of files in bundles was relative to the bundle file itself.  Make sense, I suppose.